.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ executors:
   # Whenever the Go version is updated here, .promu.yml should also be updated.
   golang:
     docker:
-      - image: cimg/go:1.19
+      - image: cimg/go:1.19.9
```

Comment:
We don't ever specify a specific version here, this tag always uses the latest point release. No need to change it.